data/wanderer/wanderers middle.txt

```diff
@@ -2853,8 +2853,23 @@ mission "Wanderers: Sestor Alt: Alnilam 2"
 		ship "Cruiser (Mark II)" "R.N.S. Dauntless"
 		ship "Cruiser (Mark II)" "R.N.S. Forthright"
 		ship "Cruiser (Mark II)" "R.N.S. Virtue"
+	npc
+		government "Navy (Oathkeeper)"
+		personality heroic
+		ship "Combat Drone" "Alpha One"
+		ship "Combat Drone" "Alpha Two"
+		ship "Combat Drone (Sidewinder Missiles)" "Alpha Three"
+		ship "Combat Drone (Sidewinder Missiles)" "Alpha Four"
+		ship "Combat Drone" "Alpha Five"
+		ship "Combat Drone" "Alpha Six"
+		ship "Combat Drone (Sidewinder Missiles)" "Alpha Seven"
+		ship "Combat Drone (Sidewinder Missiles)" "Alpha Eight"
+		ship "Combat Drone" "Alpha Nine"
+		ship "Combat Drone" "Alpha Ten"
+		ship "Combat Drone (Sidewinder Missiles)" "Alpha Eleven"
+		ship "Combat Drone (Sidewinder Missiles)" "Alpha Twelve"
```

Comment:
You could probably do this with a single variant fleet instead, although this works too.

Reply:
I might, but I rather like the irony this introduces. 
If people don't like it, I can change it.
~I'm doing it this way because I can't do that type of fleet.~